### PR TITLE
fix timeline (at least in Chrome)

### PR DESCRIPTION
**Pull request by PetrDlouhy**
In chrome the timeline is compressed on left side with no meaning. This fixes the problem.

### CODE REVIEW

File: template_profiler_panel/templates/template_profiler_panel/template.html [hunk 1]
```diff
@@ -25,8 +25,8 @@ <h4>{{ templates|length }} {% trans 'calls to Template.render()' %}</h4>
         <td>{{ template.name }}</td>
         <td class="timeline">
           <div class="djDebugTimeline">
-            <div class="djDebugLineChart" style="left: {{ template.offset_p|stringformat:'f' }}%;">
-              <strong title="Start {{ template.start }}" style="min-width: 1px; width: {{ template.rel_duration_p|stringformat:'f' }}%; background-color: {{ template.color.bg }};">&nbsp;</strong>
+            <div class="djDebugLineChart" style="margin-left: {{ template.offset_p|stringformat:'f' }}%;">
+              <div title="Start {{ template.start }}" style="min-width: 1px; width: {{ template.rel_duration_p|stringformat:'f' }}%; background-color: {{ template.color.bg }};">&nbsp;</div>
```
Comment (adamchainz) [inline template_profiler_panel/templates/template_profiler_panel/template.html:29]: why the change from `strong`?

Comment (PetrDlouhy) [reply template_profiler_panel/templates/template_profiler_panel/template.html:29]: Because on `strong` tag the parameter `width` has no effect in Chrome.

Comment (PetrDlouhy) [reply template_profiler_panel/templates/template_profiler_panel/template.html:29]: See this:
Snímek z 2021-01-12 16-59-52
Snímek z 2021-01-12 16-59-40

Comment (PetrDlouhy) [reply template_profiler_panel/templates/template_profiler_panel/template.html:29]: Now I tried also in Firefox, and the effect is the same.

**Review by adamchainz — APPROVED** [review]
Thanks for the screenshots

**Comment by adamchainz** [comment]
Released in version 2.0.2
